1.6/Dockerfile

```diff
@@ -24,31 +24,25 @@ LABEL k8s.io.description="Jenkins is a continuous integration server" \
 # 8080 for main web interface, 50000 for slave agents
 EXPOSE 8080 50000
 
-RUN curl http://pkg.jenkins-ci.org/redhat/jenkins.repo -o /etc/yum.repos.d/jenkins.repo && \
-  rpm --import http://pkg.jenkins-ci.org/redhat/jenkins-ci.org.key && \
-  yum install -y tar zip unzip java-1.7.0-openjdk jenkins && yum clean all 
-
 #RUN  curl -L -o /tmp/git.hpi http://updates.jenkins-ci.org/latest/git.hpi
 #RUN  curl -L -o /tmp/git-client.hpi http://updates.jenkins-ci.org/latest/git-client.hpi
 #RUN  curl -L -o /tmp/scm-api.hpi http://updates.jenkins-ci.org/latest/scm-api.hpi
 #RUN  curl -L -o /tmp/ssh-credentials.hpi http://updates.jenkins-ci.org/latest/ssh-credentials.hpi
 #RUN  curl -L -o /tmp/credentials.hpi http://updates.jenkins-ci.org/latest/credentials.hpi
 
-RUN  usermod -m -d "$JENKINS_HOME" jenkins && \
-  chown -R jenkins "$JENKINS_HOME" && \
-  chmod o+rw /var/lib/jenkins
+COPY run.sh /usr/local/bin/run.sh
```

Comment:
i thought i had a reason, but i think it was just a result of squashing the other run commands.  i'll move it below the run.